### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
This is how it looks like:
Screen Shot 2022-04-01 at 17 44 10

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Index.rst [hunk 1]
```diff
@@ -1,16 +1,105 @@
...
+The extension is published under GPL v2+, all included third-party libraries are
+published under their respective licenses.
+
+Authors
+-------
+
+Over a dozen contributors have been working on this area while this
+functionality was part of the TYPO3 Core. This package is now maintained by a
+loose group of TYPO3 enthusiasts inside the TYPO3 Community. Feel free to
+contact `Benni Mack`_ for any questions regarding ADOdb.
```
Comment (ohader) [inline Documentation/Index.rst]: Please avoid pointing to Benni here, refer to the issue tracker instead...

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid author pointing

File: Documentation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Pull request merged**